api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -322,6 +322,11 @@ message Bootstrap {
   // field.
   // [#not-implemented-hide:]
   map<string, core.v3.TypedExtensionConfig> certificate_provider_instances = 25;
+
+  // Specifies a set of headers that need to be registered as inline header. This configuration
+  // allows users to customize the inline headers on-demand at Envoy startup without modifying
+  // Envoy's source code.
+  repeated CustomInlineHeader inline_headers = 32;
```

Comment:
Worth calling out that inline headers will be elided?  If I recall correctly
 
foo: bar
foo: eep

will turn into

foo: bar, eep

Reply:
please call out the new field in docs/root/version_history/current.rst

Reply:
get it.